inspirehep/modules/pidstore/minters.py

```diff
@@ -41,5 +41,5 @@ def inspire_recid_minter(record_uuid, data):
     if 'control_number' in data:
         args['pid_value'] = data['control_number']
     provider = InspireRecordIdProvider.create(**args)
-    data['control_number'] = str(provider.pid.pid_value)
+    data['control_number'] = provider.pid.pid_value
```

Comment:
which seems to be our fault :) https://github.com/inspirehep/inspire-next/pull/1744#issuecomment-260725528